### PR TITLE
Fix ansible-lint pipeline

### PR DESCRIPTION
**Pull request by ra-beer**
Signed-off-by: Ramona Beermann <ramona.beermann@osism.tech>

### CODE REVIEW

File: roles/hosts/tasks/type-block.yml [hunk 1]
```diff
@@ -1,5 +1,5 @@
 ---
-- name: Get /etc/hosts file
+- name: Get /etc/hosts file  # noqa var-naming[no-reserved]
   ansible.builtin.slurp:
     src: "{{ hosts_file }}"
   register: hosts
```
Comment (osfrickler) [inline roles/hosts/tasks/type-block.yml:5]: I'd prefer to actually rename this var, maybe `hosts_entries`? or `hosts_file_contents`?

Comment (ra-beer) [reply roles/hosts/tasks/type-block.yml:5]: Sure can do this. If it is not necessary that the var is named "hosts"

File: roles/hosts/tasks/type-block.yml [hunk 2]
```diff
@@ -2,7 +2,7 @@
 - name: Get /etc/hosts file
   ansible.builtin.slurp:
     src: "{{ hosts_file }}"
-  register: hosts
+  register: hosts_entries
```
Comment (osfrickler) [inline roles/hosts/tasks/type-block.yml:5]: you'll also have to change the references to this variable below. I'm surprised that the test is still passing, seems like the coverage could be improved

**Review by osfrickler — CHANGES_REQUESTED** [review]

File: roles/hosts/tasks/type-block.yml (hunk 2 above)
Comment (ra-beer) [reply roles/hosts/tasks/type-block.yml:5]: Something went wrong with my push force. Now it should be fixed.

**Pull request merged**